### DISCUSSION TITLE
Dropdowns not working

### DISCUSSION THREAD

**Discussion by fazal776** [twbs/bootstrap#40494] [Q&A]
Though i have included all the script links in the starter template still my dropdowns are not working.

**Comment by julien-deramond** [comment]
Hi @fazal776,
Could you share some code or a reproducible environment with CodePen, Stackblitz, or any other tools? Otherwise, it won't be possible to help :)

**Comment by fazal776** [comment]
Hi Julien,

I have attached the index.html file with this mail. I was making a navbar
using bootstrap. I copied the starter template and also included the popper
and other script links in it. However in the navbar there is a
dropdown button which is not working when i am clicking on it (means it
does not show the list elements in the code).

Thank you very much,
Regards,
Fazal

**Reply by julien-deramond** [reply]
The `index.html` file is not available in this GitHub Discussion. I'm not sure if it's possible to do this via your email client.
